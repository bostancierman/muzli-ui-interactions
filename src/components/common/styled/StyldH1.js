import styled from 'styled-components'
import { map } from 'react-immutable-proptypes'
import { Map } from 'immutable'

export const StyldH1 = styled.h1`
  ${props => props.css.toJS()}
`

StyldH1.defaultProps = {
  css: Map()
}

StyldH1.propTypes = {
  css: map
}

export default StyldH1
