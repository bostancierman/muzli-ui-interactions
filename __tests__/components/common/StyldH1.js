import React from 'react'
import renderer from 'react-test-renderer'
import StyldH1 from 'src/components/common/styled/StyldH1.js'

it('renders correctly', () => {
  expect(<StyldH1>Boilerplate</StyldH1>).toMatchSnapshot()
})