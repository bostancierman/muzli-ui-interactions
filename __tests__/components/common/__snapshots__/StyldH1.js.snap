// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<styled.h1
  css={Immutable.Map {}}
>
  Boilerplate
</styled.h1>
`;
